drag_and_drop_v2/translations/eo/LC_MESSAGES/text.po

```diff
@@ -0,0 +1,547 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2016-03-30 16:54+0500\n"
+"PO-Revision-Date: 2016-03-30 16:54+0500\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: en\n"
```

Comment:
I created this file with the help of [i18n-tool](https://github.com/edx/i18n-tools). I ran `i18n_tool dummy` command. Yes it should be `eo`